testsuite/integration/basic/src/test/java/org/wildfly/test/integration/observability/opentelemetry/application/TestLibraries.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2022 Contributors to the Eclipse Foundation
```

Comment:
```Eclipse Foundation``` intended here?

Reply:
I forked/copied this class, so that's correct.